orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -198,21 +198,21 @@
              {% if 'ATLAS' in exposed and 'ATLAS_SERVER' in salt['pillar.get']('gateway:location') -%}
              <param>
                  <name>ATLAS</name>
-                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySessions=true</value>
```

Comment:
Is this enabled for all runtime versions? Is there a way to enable this only for 7.2.7 and above?

Reply:
All run times, knox should ignore it if it's not supported.  There's no way to make it 7.2.7+ only.